lerobot/common/datasets/push_dataset_to_hub/oxe/configs.yaml

```diff
@@ -0,0 +1,778 @@
+StateEncoding:
+  NONE: -1               # No Proprioceptive State
+  POS_EULER: 1           # EEF XYZ (3) + Roll-Pitch-Yaw (3) + <PAD> (1) + Gripper Open/Close (1)
+  POS_QUAT: 2            # EEF XYZ (3) + Quaternion (4) + Gripper Open/Close (1)
+  JOINT: 3               # Joint Angles (7, <PAD> if fewer) + Gripper Open/Close (1)
+  JOINT_BIMANUAL: 4      # Joint Angles (2 x [ Joint Angles (6) + Gripper Open/Close (1) ])
```

Comment:
This `JOINT_BIMANUAL` is intended to be used only for the table-top aloha (not used in this conversion config). We can remove/comment this line too to prevent confusion too.